qiime2/plugins.py

```diff
@@ -37,6 +37,13 @@ def _init_metadata_(self, ref, factory):
         self._init_data_refs[ref] = factory
         return ref
 
+    def _init_data_collection_(self, ref, collection_type, *records):
+        self._init_data_refs[ref] = ref
+        refs = sorted([i.ref for i in records])
+        t = ', '.join(refs).strip(',')
```

Comment:
The `strip` should be unnecessary.

```suggestion
        t = ', '.join(refs)
```

Reply:
Hmmm, weird.  I had a test case failing that showed me the `strip` was necessary, but I'm sure what happened.  Fixed!